Sources/GRPCHTTP2Core/GRPCStreamStateMachine.swift

```diff
@@ -724,7 +742,7 @@ extension GRPCStreamStateMachine {
     }
 
     let statusMessage =
-      metadata.firstString(forKey: .grpcStatusMessage)
+      metadata.first(name: GRPCHTTP2Keys.grpcStatusMessage.rawValue)
```

Comment:
Why don't we allow `firstString(forKey:)` to take a `canonicalForm` param and just pass that down instead? Seems a shame to drop back to using the verbose names.